packages/fluentui/docs/src/prototypes/table/SelectableTable.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  gridCellWithFocusableElementBehavior,
+  gridNestedBehavior,
+  gridCellBehavior,
+  gridRowBehavior,
+} from '@fluentui/accessibility';
+import { Table, Checkbox, Box } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const SelectableTable = () => {
+  const rows = [
+    ['Test name 1', 'General', 'Office'],
+    ['Test name 2', 'Commander', 'Office'],
+    ['Test name 3', 'Other', 'Field'],
+  ];
+
+  const rowsLength = rows.length;
+
+  const [selectedRows, setSelectedRows] = React.useState(Array(rowsLength).fill(false));
```

Comment:
Agreed with @layershifter that it's better to use useReducer here, will update it